src/features/settings/settingsSlice.ts

```diff
@@ -16,18 +17,48 @@ export interface SettingsState {
   explorerLiquidAPI: string;
   explorerLiquidUI: string;
   assets: Asset[];
+  baseUrl: string;
+  useProxy: boolean;
   marketsLabelled?: MarketLabelled[];
-  tdexDaemonBaseUrl: string;
   macaroonCredentials?: string;
   tdexdConnectUrl?: string;
+  proxyCommand?: Child;
 }
 
+export const startProxy = createAsyncThunk<Child, void, { state: RootState }>(
+  'settings/startProxy',
+  async (_, thunkAPI) => {
+    const { settings } = thunkAPI.getState();
+    console.log('HEYHEY', settings);
+    if (settings.proxyCommand) {
+      console.log('stopping proxy');
+      settings.proxyCommand.kill();
+    }
+    console.log('starting up proxy with url', settings.tdexdConnectUrl);
+    const cmd = await Command.sidecar('grpcproxy', ['--tdexdconnecturl', settings.tdexdConnectUrl!]).spawn();
+    console.log('done!!!', cmd.pid);
+    return cmd;
+  }
+);
+
+export const stopProxy = createAsyncThunk<void, void, { state: RootState }>(
+  'settings/stopProxy',
+  async (_, thunkAPI) => {
+    const { settings } = thunkAPI.getState();
+    if (settings.proxyCommand) {
+      settings.proxyCommand.kill();
+    }
+    return;
+  }
+);
+
 export const initialState: SettingsState = {
-  chain: network.chain,
-  explorerLiquidAPI: network.explorerLiquidAPI,
-  explorerLiquidUI: network.explorerLiquidUI,
-  tdexDaemonBaseUrl: network.tdexDaemonBaseUrl,
+  chain: 'regtest',
+  explorerLiquidAPI: 'https://blockstream.info/liquid/api',
+  explorerLiquidUI: 'https://blockstream.info/liquid',
+  baseUrl: '__TAURI__' in window ? 'http://localhost:3030' : 'http://localhost:9000',
```

Comment:
the baseUrl if not in tauri should come from the tdexdconnect url when we extract. 

If not possible at init, better to leave it empty, will be replaced by the UI